File: Exercicio_17/biblioteca/src/entities/LivroFisico.java
package entities;

public class LivroFisico extends Livro{

	private int numPaginas;
	
	public LivroFisico(String titulo, String autor, int anoPublicacao, int numPaginas) {
		super(titulo, autor, anoPublicacao);
		this.numPaginas = numPaginas;
	}

	@Override
	public void exibirInfo() {
		super.exibirInfo();
		System.out.println("Número de Páginas: " + numPaginas);
	}
	
}

File: 17_OrientacaoAObjetos/LivroFisico.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace _17_OrientacaoAObjetos
{
    internal class LivroFisico : Livro
    {
        public int NumeroDePaginas { get; set; }

        public override void ExibirInfo()
        {
            Console.WriteLine("\n=== Livro Físico ===");
            base.ExibirInfo();
            Console.WriteLine("Número de páginas: " + NumeroDePaginas);
        }
    }
}
